### ISSUE TITLE
Error BufReadPost Autocommands after git commit command

### ISSUE THREAD

**Issue by jrmejiaa** [nvim-lua/kickstart.nvim#1050]
## Describe the bug

Hi devs, 

I am using the last version commit 56b9114 with Neovim `v0.10.1`. I got the following error after trying to commit a change in my git

```bash
# git commit
Error detected while processing BufReadPost Autocommands for "*":
Error executing lua callback: /usr/local/share/nvim/runtime/filetype.lua:35: Error executing lua: /usr/local/share/nvim/runtime/filetype.lua:36: BufReadPost Autocommands for "*"..FileType Autocommands for "*": Vim(append):Error executing lua callback: ...eesitter/lua/nvim-treesitte
r/shell_command_selectors.lua:308: bad argument #3 to 'format' (value expected)
stack traceback:
        [C]: in function 'format'
        ...eesitter/lua/nvim-treesitter/shell_command_selectors.lua:308: in function 'select_download_commands'
        ...vim/lazy/nvim-treesitter/lua/nvim-treesitter/install.lua:406: in function 'run_install'
        ...vim/lazy/nvim-treesitter/lua/nvim-treesitter/install.lua:495: in function 'install_lang'
        ...vim/lazy/nvim-treesitter/lua/nvim-treesitter/install.lua:552: in function <...vim/lazy/nvim-treesitter/lua/nvim-treesitter/install.lua:514>
        ...vim/lazy/nvim-treesitter/lua/nvim-treesitter/install.lua:564: in function <...vim/lazy/nvim-treesitter/lua/nvim-treesitter/install.lua:561>
        [C]: in function 'nvim_cmd'
        /usr/local/share/nvim/runtime/filetype.lua:36: in function </usr/local/share/nvim/runtime/filetype.lua:35>
        [C]: in function 'nvim_buf_call'
        /usr/local/share/nvim/runtime/filetype.lua:35: in function </usr/local/share/nvim/runtime/filetype.lua:10>
stack traceback:
        [C]: in function 'nvim_cmd'
        /usr/local/share/nvim/runtime/filetype.lua:36: in function </usr/local/share/nvim/runtime/filetype.lua:35>
        [C]: in function 'nvim_buf_call'
        /usr/local/share/nvim/runtime/filetype.lua:35: in function </usr/local/share/nvim/runtime/filetype.lua:10>
stack traceback:
        [C]: in function 'nvim_buf_call'
        /usr/local/share/nvim/runtime/filetype.lua:35: in function </usr/local/share/nvim/runtime/filetype.lua:10>
```

## To Reproduce
- Set `nvim` as default editor in your shell
- Try to make a commit in any repository with the command `git commit` (without `-m` flag)
- Got error

## Desktop

- OS: Debian 12
- Terminal: zsh 5.9 (x86_64-debian-linux-gnu)

## Neovim Version
```
NVIM v0.10.1
Build type: Release
LuaJIT 2.1.1713484068
```

**Comment by jrmejiaa** [comment]
This was a problem of `nvim-treesitter`, this issue helped me https://github.com/nvim-treesitter/nvim-treesitter/issues/5353

If I run the command `:TSInstall gitcommit`. I don't get the error. Is it well known this behavior?

**Comment by iton0** [comment]
I believe this issue is being addressed in #1061

**Comment by feoh** [comment]
Just merged that PR. Closing.
